app/src/main/java/me/ccrama/redditslide/util/SortingUtil.java

```diff
@@ -0,0 +1,212 @@
+package me.ccrama.redditslide.util;
+
+import android.graphics.Typeface;
+import android.text.Spannable;
+import android.text.SpannableString;
+import android.text.style.ForegroundColorSpan;
+import android.text.style.StyleSpan;
+
+import net.dean.jraw.paginators.Sorting;
+import net.dean.jraw.paginators.SubmissionSearchPaginator;
+import net.dean.jraw.paginators.TimePeriod;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Locale;
+import java.util.Map;
+
+import me.ccrama.redditslide.Activities.Search;
+import me.ccrama.redditslide.ColorPreferences;
+import me.ccrama.redditslide.R;
+import me.ccrama.redditslide.Reddit;
+
+public class SortingUtil {
```

Comment:
I am pretty sure I touched no logic in any of these methods. I just straight copy and pasted out and replaced usages from `Reddit.` to `SortingUtil.`

I did include usages of the god context where the activity context was not necessary.

Reply:
I really like this! Have no idea what possessed me to make this part of Reddit